concepticondata/concepticon.tsv

```diff
@@ -3978,3 +3978,4 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3992	SARDINE	Animals	A pilchard or other fish in the herring family.	Person/Thing	
 3993	JINGLE (INSTRUMENT)	Speech and language	One of a cluster of small bells, shaken or tapped on the palm of the player's hand. The small metal discs arranged around the frame of a tambourine are also called "jingles".	Person/Thing	
 3994	PIANO (INSTRUMENT)	Speech and language	A large musical instrument with a row of black and white keys that are pressed to play notes.	Person/Thing
+3995	AMAZONIAN MANATEE	Animals	Trichechus inunguis ()GBF ID: 2435296), one of the four living Manatee species. Native to the Amazon river system, the amazonian manatee is a fully aquatic mammal.	Person/Thing	
```

Comment:
There is one bracket too much and could you also add the link to the ID?

Reply:
Sure, I will change this